src/analysis/interpolation/Bezier3D.h

```diff
@@ -20,45 +20,76 @@
 #include "ParametricLine.h"
 #include "qgslogger.h"
 
-/** \ingroup analysis
+/**
+ * \ingroup analysis
  * Class Bezier3D represents a bezier curve, represented by control points. Parameter t is running from 0 to 1. The class is capable to calculate the curve point and the first two derivatives belonging to t.*/
 class ANALYSIS_EXPORT Bezier3D: public ParametricLine
 {
   protected:
 
   public:
-    /** Default constructor*/
+
+    /**
+     * Default constructor*/
```

Comment:
IMHO a wast of vertical space.  One line comments should stay on one line. OTOH if we put `/**` on a separate line the `*/` should also be on a separate line.